main/main.go

```diff
@@ -73,11 +77,15 @@ var (
 func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 	utilruntime.Must(lcav1alpha1.AddToScheme(scheme))
+	utilruntime.Must(seedgenv1alpha1.AddToScheme(scheme))
 	utilruntime.Must(ocpV1.AddToScheme(scheme))
 	utilruntime.Must(mcv1.AddToScheme(scheme))
 	utilruntime.Must(velerov1.AddToScheme(scheme))
 	utilruntime.Must(operatorsv1alpha1.AddToScheme(scheme))
 	utilruntime.Must(operatorv1alpha1.AddToScheme(scheme))
+	utilruntime.Must(clusterv1.AddToScheme(scheme))
```

Comment:
?
```suggestion
	utilruntime.Must(clusterv1.Install(scheme))
```

AddToScheme seems deprecated for the clusterv1 package - https://github.com/openshift-kni/lifecycle-agent/blob/main/vendor/open-cluster-management.io/api/cluster/v1/register.go#L9-L22

(also deprecated for the ocpV1 package but not touched by this PR, in case's useful)

Reply:
Given that the controller-gen we're using doesn't add an "Install", I'd prefer to keep this as-is for now until we change others, since AddToScheme and Install are pointing to the same function.